packages/select/src/components/omnibar/omnibar.tsx

```diff
@@ -107,8 +107,7 @@ export class Omnibar<T> extends React.PureComponent<IOmnibarProps<T>> {
     };
 
     private handleOverlayClose = (event?: React.SyntheticEvent<HTMLElement>) => {
-        const { overlayProps = {} } = this.props;
-        Utils.safeInvoke(overlayProps.onClose, event);
+        Utils.safeInvokeMember(this.props.overlayProps, "onClose", event);
```

Comment:
why is this better? it seems like the other syntax at least helped with typos and autocomplete

Reply:
this also helps with the same things! `safeInvokeMember` is fully typesafe, including the arguments. the big win is you don't need to deconstruct&default.

https://github.com/palantir/blueprint/blob/develop/packages/core/src/common/utils/safeInvokeMember.ts

Reply:
ok, seems fine